src/extension/ip_extraction.c

```diff
@@ -33,28 +33,28 @@ static zend_string *nonnull _x_forwarded_for_key, *nonnull _x_real_ip_key,
     *nonnull _forwarded_key, *nonnull _via_key, *nonnull _true_client_ip_key,
     *nonnull _remote_addr_key;
 
-static THREAD_LOCAL_ON_ZTS zend_string *nullable _ipheader;
+static THREAD_LOCAL_ON_ZTS zend_string *nullable _client_ip_header;
 
 typedef bool (*extract_func_t)(zend_string *nonnull value, ipaddr *nonnull out);
 
-static ZEND_INI_MH(_on_update_ipheader);
+static ZEND_INI_MH(_on_update_client_ip_header);
 
 // clang-format off
 static const dd_ini_setting ini_settings[] = {
-    DD_INI_ENV("ipheader", "", PHP_INI_SYSTEM, _on_update_ipheader),
+    DD_TRACE_INI_ENV("client_ip_header", "", PHP_INI_SYSTEM, _on_update_client_ip_header),
```

Comment:
Removing an ini setting could be a breaking change, I'd suggest keeping both and if they are both present give priority to the tracer one.

Reply:
Did we document ipheader anywhere? I thought it was used only for testing purposes

Reply:
Good point, it used to be meant for customers but I guess we might not have documented it. Please check the docs and if it isn't there then it should be fine to remove it.

Reply:
Can't find any reference to it on the Documentation reporsitory